WalletWasabi/WebClients/Wasabi/HttpClientFactory.cs

```diff
@@ -20,6 +21,15 @@ public class HttpClientFactory
 		/// <param name="torEndPoint">If <c>null</c> then clearnet (not over Tor) is used, otherwise HTTP requests are routed through provided Tor endpoint.</param>
 		public HttpClientFactory(EndPoint? torEndPoint, Func<Uri> backendUriGetter)
 		{
+			SocketHandler = new SocketsHttpHandler()
```

Comment:
This shouldn't be a property. In case you did it in this way to be able to dispose it then use the `HttpClient::ctor(HttpMessageHandler handler, bool disposeHandler)`

Reply:
Normally, I would have done that in your way but this way it fixes CA2000 warning.

If we don't want that property, then the code will like this:

```csharp
#pragma warning disable CA2000 // Dispose objects before losing scope
			SocketsHttpHandler socketHandler = new()
			{
				// Only GZip is currently used by Wasabi Backend.
				AutomaticDecompression = DecompressionMethods.GZip,
				PooledConnectionLifetime = TimeSpan.FromMinutes(5)
			};
#pragma warning restore CA2000 // Dispose objects before losing scope
```

Reply:
I prefer the one with suppression and no property.

Reply:
I am fine with both.

Reply:
@lontivero Do you prefer that `#pragma` solution?

Reply:
This is one of those situations where all the options suck bad. Let it as it is.